src/store/hey.ts

```diff
@@ -75,7 +75,7 @@ export const pendingTxsAtom = atomWithQuery<Heystack[], string>(get => ({
       )
       .map(tx => tx.tx_id);
     const final = await Promise.all(
-      heyTxs.map(async txid => client.getTransactionById({ txId: txid }))
+      heyTxs.map(async txid => await client.getTransactionById({ txId: txid }))
```

Comment:
Not necessary when constructing a Promise.all
```suggestion
      heyTxs.map(txid => client.getTransactionById({ txId: txid }))
```

Reply:
My linter forced me to add the `await` when I rebased but overlooked just removing the `async`, thx!